baseband/dada/base.py

```diff
@@ -430,19 +430,31 @@ def open(name, mode='rs', subset=None, header=None, **kwargs):
     is_template = isinstance(name, six.string_types) and ('{' in name and
                                                           '}' in name)
     is_sequence = isinstance(name, (tuple, list))
+    is_sequentialfile = isinstance(name, (sf.SequentialFileReader,
```

Comment:
I'm wondering if it wouldn't be cleaner to move it right up front (since the whole purpose of this is to create a sequential file, it seems logical to just start by checking whether one has one already), i.e., 
```
if isinstance(name, sf.SequentialFileBase):
    assert ('w' in mode and name.mode == 'w+m' or
               'r' in mode and name.mode == 'rb'), "...
    ...
else:
    is_template = ...
```

Reply:
Probably, will do that.